test/e2e/ibmcloud_test.go

```diff
@@ -127,6 +132,248 @@ func TestCreatePeerPodAndCheckEnvVariableLogsWithImageAndDeployment(t *testing.T
 	doTestCreatePeerPodAndCheckEnvVariableLogsWithImageAndDeployment(t, assert)
 }
 
+func TestCreatePeerPodWithPVC(t *testing.T) {
+	if os.Getenv("TEST_CSI_WRAPPER") == "yes" {
+		assert := IBMCloudAssert{
+			vpc: pv.IBMCloudProps.VPC,
+		}
+		nameSpace := "kube-system"
+		pvcName := "my-pvc"
+		mountPath := "/mount-path"
+		storageClassName := "ibmc-vpc-block-5iops-tier"
+		storageSize := "10Gi"
+		podName := "nginx-pvc-pod"
+		imageName := "nginx"
+		csiContainerName := "ibm-vpc-block-podvm-node-driver"
+		csiImageName := "gcr.io/k8s-staging-cloud-provider-ibm/ibm-vpc-block-csi-driver:master"
+
+		myPVC := newPVC(nameSpace, pvcName, storageClassName, storageSize, corev1.ReadWriteOnce)
+		myPodwithPVC := newPodWithPVCFromIBMVPCBlockDriver(nameSpace, podName, imageName, imageName, csiContainerName, csiImageName, withRestartPolicy(corev1.RestartPolicyNever), withPVCBinding(mountPath, pvcName))
+		doTestCreatePeerPodWithPVCAndCSIWrapper(t, assert, myPVC, myPodwithPVC, mountPath)
+	} else {
+		log.Infof("Ignore PeerPod with PVC (CSI wrapper) test")
+	}
+}
+
+func newPodWithPVCFromIBMVPCBlockDriver(namespace, podName, containerName, imageName, csiContainerName, csiImageName string, options ...podOption) *corev1.Pod {
```

Comment:
Would it make more sense to make a new `podOption` function `withPVCFromIBMVPCBlockDriver` as the IBM VPC Block driver PVC should be an option that can be attached to an existing pod

Reply:
It's not simple to add PVC with new `podOption`:
- attach PVC to an existing pod is easily we can just add new entry of `volume` and `volumeMounts`. 
- we also need attach `ibm-vpc-block-podvm-node-driver` container and `csi-podvm-wrapper` container, and the `ibm-vpc-block-podvm-node-driver` container need some special volumeMounts.

We need think about how we append the `ibm-vpc-block-podvm-node-driver` container and `csi-podvm-wrapper` container automatically in future pr but not in this one.

Reply:
Yeah, I think we'll add this in Pod descriptor via webhook, the webhook might handle it per cloud provider like: ibmcloud, aws, azure etc.